### ISSUE TITLE
Support Bootstrap icons

### ISSUE THREAD

**Issue by PanderMusubi** [Xaviju/inkscape-open-symbols#114]
Please, support Bootstrap icons

https://icons.getbootstrap.com/

Their license is MIT

**Comment by denilsonsa** [comment]
This seems to be obsolete, as we already have Bootstrap. Does it need to be updated? https://github.com/PanderMusubi/inkscape-open-symbols/tree/master/Bootstrap

**Comment by PanderMusubi** [comment]
Indeed. Yes, it needs to be updated. Renaming the title of this issue.
